lib/indexMealTables.js

```diff
@@ -0,0 +1,40 @@
+var $ = require('jQuery');
+var foods = require('./foods.js')
+
+function createTables(meals) {
+  meals.forEach(function(meal) {
+    createMealTable(meal, meal.name, mealString)
+  })
+}
+
+function createMealTable(meal, name, mealString) {
+  meal.foods.forEach(function(food) {
+    $(`#${name.toLowerCase()}-table tbody`).append(mealString(meal, food))
+  })
+}
+
+function mealString(meal, food) {
```

Comment:
Much of this page, as well as foods.js, will alter with my pending PR and todays in-class partner assignment, if the merge gets messy, we may have to work through it tomorrow.